src/file.toit

```diff
@@ -45,61 +46,126 @@ class Stream implements Reader:
 
   constructor.internal_ .fd_:
 
-  // Open a file for reading.
-  constructor.for_read path:
+  /**
```

Comment:
Most functions only have their toitdocs changed, and I often changed the "filename"/"name" to "path".
Also added some types when I could.